### ISSUE TITLE
`send` method send entire arraybuffer instead of given slice.

### ISSUE THREAD

**Issue by hackwaly** [feross/chrome-dgram#3]
I use this to do dns query, but encouter strange bug.
It spend me a half night to find out it.

https://github.com/feross/chrome-dgram/blob/master/index.js#L246

I solved it by just send the slice.
```javascript
  chrome.sockets.udp.send(self.id, buffer.buffer.slice(offset, length) /* buffer.toArrayBuffer() is slower */,
                      address, +port, function (sendInfo) {
    if (sendInfo.resultCode < 0) {
      var err = new Error('Socket ' + self.id + ' send error ' + sendInfo.resultCode)
      callback(err)
      self.emit('error', err)
    } else {
      callback(null)
    }
  })
```

**Comment by feross** [comment]
Thanks for reporting this. Fixed and released as v2.0.6

**Comment by hackwaly** [comment]
Maybe directly use `.buffer` is not correct also?
I have not read the implementation of buffer module on npm. Should it consider byteOffset and byteLength properties?
